polkadot-parachains/Cargo.toml

```diff
@@ -19,7 +19,7 @@ hex-literal = "0.2.1"
 async-trait = "0.1.42"
 
 # Parachain runtimes
-rococo-parachain-runtime = { package = "rococo-runtime", path = "rococo" }
+rococo-parachain-runtime = { package = "rococo-parachain-runtime", path = "rococo" }
```

Comment:
the rename is now obsolete
(`package = "rococo-parachain-runtime", `) part

Reply:
What do you mean? I don't understand, sorry

Reply:
I means you can remove `package = ..` part now